core/src/main/java/org/elasticsearch/action/get/MultiGetRequest.java

```diff
@@ -123,6 +129,11 @@ public String id() {
             return this.id;
         }
 
+        public Item id(String id) {
```

Comment:
Where is this setter needed? I experimentally removed it and everything seems fine, in which case I would like to remove it.

Reply:
Yes, this can be removed. This was a left over from the parser.